docs/posts/gallery.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Gallery
+subtitle: Features & examples
+layout: default
+---
+
+# Features
+
+### Flexible selections, coloring, and differential expression of your selected sets of cells
+<img src="../images/diffexp.gif" width="600"/>
+
+### Single-gene analyses (e.g. expression analysis)
+<img src="../images/customGene.gif" width="600" />
+
```

Comment:
NOTE: I will plan to add a GIF showing how user can change embeddings (assuming they are loaded in with the H5AD).

Reply:
Also plan to add a GIF showing color-by of a category and how it creates the colored charts per label when a different category is open

Reply:
Possibly will also make a GIF showing color-by of a gene, and the mini-histograms per label when a category is open